panoramisk/testing.py

```diff
@@ -45,14 +45,14 @@ class Manager(manager.Manager):
 
     def __init__(self, **config):
         self.defaults.update(
-            connection_class=Connection,
```

Comment:
I'm not an expert, however, at least to me, I think it's the class used, not the factory that generates an handler.
For me, the previous name was good.